persistence-utils-demo-webtest/src_test/com/axonivy/utils/persistence/webtest/WebTestDemoIT.java

```diff
@@ -183,12 +182,8 @@ private void startProcessPersonSearch() {
 		open(EngineUrl.createProcessUrl("persistence-utils-demo/173A4BC5D38BAD52/personSearch.ivp"));
 	}
 
-	private static void startLoginAsAdmin() {
-		open(EngineUrl.base() + "default-workflow/faces/login.xhtml");
-
-		$(By.id("loginForm:userName")).shouldBe(enabled).sendKeys("jpa_admin");
-		$(By.id("loginForm:password")).shouldBe(enabled).sendKeys("jpa_admin");
-		$(By.id("loginForm:login")).shouldBe(enabled).click();

```

Comment:
from 10.0.14 onwards we have an official api to do the login.
so we are responsible to keep it stable among ivy-versions.